### DISCUSSION TITLE
[ Choix du statut ] Différences entre « entreprise individuelle » et « auto-entrepreneur »

### DISCUSSION THREAD

**Discussion by glopezrios** [betagouv/mon-entreprise#2951] [Idées]
Dans le simulateur « choix du statut », il y a notamment comme statuts possibles « auto-entrepreneur » et « entreprise individuelle ». Or au sens strict, l’auto-entreprise est une forme d’entreprise individuelle (qui bénéficie des régimes micro-social et micro-fiscal).

Or cela pose un problème d’orientation pour les usagers, notamment lors de la création de leur entreprise sur le guichet unique. Pour créer une micro-entreprise, il faut d’abord choisir « entrepreneur individuel » pour accéder ensuite à la question sur le statut de micro-entrepreneur (cf. capture d’écran).

On pourrait d’ailleurs faire la même remarque pour [la page de contact de l’URSSAF](https://www.urssaf.fr/accueil/contacter-urssaf.html) : il faut d’abord cliquer sur « indépendant » pour ensuite avoir accès à « auto-entrepreneur ».

Une solution serait peut-être, sur le simulateur de « choix du statut », de préciser le statut « entreprise individuelle » en précisant « hors auto-entrepreneur » ?

La question pourrait aussi se poser pour le simulateur « entreprise individuelle », qui  intègre le régime micro-fiscal mais pas le régime micro-social (et donc exclut là aussi l’auto-entrepreneur).
![Capture d’écran 2024-05-14 171005](https://github.com/betagouv/mon-entreprise/assets/168096444/0c98bdfb-41da-4e29-aa7d-0b189466e544)

**Comment by johangirod** [comment]
Relié #2159

**Comment by glopezrios** [comment]
L'issue #2159 me semble très pertinente et a en effet un lien avec cette issue. 

Mais cette issue pose la question du titre même du simulateur pour "entreprise individuelle" ou tout du moins du besoin éventuel d'une phrase introductive qui indique que ce simulateur ne traite pas le cas de l'auto-entreprise. Il me semble que c'est un peu différent que #2159 mais si cela traite de la même question.

Et cette issue est aussi en "lien logique" avec #3011  

En bref:
- l'auto-entreprise (ou micro-entreprise) est une forme d'entreprise individuelle
- il ne faut pas confondre "micro-fiscal" et "micro-entreprise".

**Comment by glopezrios** [comment]
A prolonger dans #3018
